src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -98,6 +99,10 @@
 
     private final ISecondFactorAuthenticator secondFactorManager;
 
+    private final int USER_NAME_MAX_LENGTH = 255;
```

Comment:
I know it wasn't in the request but I think it would be a minor change to also add a `USER_NAME_MIN_LENGTH = 1;` and have it greater than or equal to that too? We do the front end 'this must have a value' check but nothing back end too...

Reply:
I think we should enforce that with a "not `.isEmpty()` check, rather than a hardcoded 1 value!

Reply:
Yes, that would be far nicer wouldn't it, d'oh

Reply:
Good point - I'll check for an empty name which should be equivalent.